src/pages/Location/Location.js

```diff
@@ -41,13 +40,12 @@ const LocationComponent = () => {
   const dispatchRedux = useDispatch();
   const [selectedZone, setselectedZone] = useState();
   const [selectedToEditSite, setSelectedToEditSite] = useState({});
-  // const [selectedRegion, setSelectedRegion] = useState({});
   const [handleConfirm, setHandleConfirm] = useState(false);
+  const [locZipCode, setLocZipCode] = useState();
```

Comment:
could you rename to just zipCode since all the redux vars end with `Redux`?

Reply:
zipCode is one of the properties of selectedToEdit that is destructured in the useEffect on line 98.  I added the loc to the front so we didn't have 2 consts with the same name.  and since its the location component and its the component state I choose loc.

Reply:
ok that makes sense